BrowserServicesKit/Sources/Suggestions/ScoringService.swift

```diff
@@ -19,33 +19,33 @@
 import Foundation
 import Common
 
-typealias Score = Int
+struct ScoringService  {
 
-extension Score {
-
-    // swiftlint:disable:next cyclomatic_complexity
-    init(title: String?, url: URL, visitCount: Int, query: Query, queryTokens: [Query]? = nil) {
+    /// Scores a suggestion based on the query and the suggestion's title and url.
+    static func score(title: String?, url: URL, visitCount: Int = 0, lowerQuery: String, queryTokens: [String]? = nil) -> Int { // swiftlint:disable:this cyclomatic_complexity
```

Comment:
What does lowerQuery mean? Is it lowercased?

Reply:
added some docs